assemble/bin/accumulo

```diff
@@ -58,7 +58,30 @@ function main() {
     echo "$CLASSPATH"
     exit 0
   fi
+  
+  # Set up path variable for default import config file
+  export jShellPath="$conf/jshell-init.jsh"
+ 
+  # Verify what file(s) is loaded into JShell
+  if [[ $cmd == "jshell" ]] && [[ "$#" -eq 1 ]]; then
+    # Load in default Accumulo APIs into JShell
+    CLASSPATH="$CLASSPATH" "$@" "$jShellPath" 
+    exit 0
 
+  elif [[ $cmd == "jshell" ]] && [[ "$#" -ne 1 ]]; then
+    cmdArr=( "$@" )
+    # Traverse through user bash command and verify -startup argument exists
+    for arg in "${!cmdArr[@]}"; do
+       if [[ "${cmdArr[arg]}" =~ "-startup" ]]; then
+         # Insert "DEFAULT" JShell script after -startup argument
+         cmdArr=( "${cmdArr[@]:0:arg}" "DEFAULT" "${cmdArr[@]:arg+1}" )
+       fi
+    done
+    # Load in default java libraries first followed by custom file(s)
+    CLASSPATH="$CLASSPATH" "${cmdArr[@]}"
+    exit 0
+  fi
```

Comment:
CLASSPATH is probably already exported, so you might not need that part. If you do, then `export CLASSPATH` on the preceding line before the exec might be more clear.

Also, you can probably inject the startup file (if it exists) before the user's own arguments, as in:

```suggestion
  jShellPath="$conf/jshell-init.jsh"
  if [[ $cmd == "jshell" ]]; then
    shift
    if [[ -f "$jShellPath" ]]; then
      exec "$cmd" --startup "$jShellPath" "$@"
    else
      exec "$cmd" "$@"
    fi
  fi
```